src/userobjects/MoabSkinner.C

```diff
@@ -280,15 +280,12 @@ void
 MoabSkinner::update()
 {
   _console << "Skinning geometry into " << _n_temperature_bins << " temperature bins, "
-           << _n_density_bins << " density bins, and " << _n_block_bins << " block bins... ";
+           << _n_density_bins << " density bins, and " << _n_block_bins << " block bins... "
+           << std::endl;
 
   // Clear MOAB mesh data from last timestep
   reset();
 
-  if (_use_displaced)
```

Comment:
ok, this one does have to be here -- because we do technically allow `MoabSkinner` to run without an `OpenMCCellAverageProblem`. But we do not need it in the tally objects because those cannot exist without an `OpenMCCellAverageProblem`. Just noting as FYI